src/components/Header/Header.css

```diff
@@ -1,13 +1,17 @@
-.Header{
-	overflow: auto;
-}
-.Header.dark{
-	background: #333;
+.Header {
+	align-items: center;
```

Comment:
Isn't this just for ``flexbox`` stuff?

Reply:
Yes forgot to remove this when removing flex from the build